src/lib/pickles/common.mli

```diff
@@ -38,16 +38,15 @@ val finite_exn : 'a Kimchi_types.or_infinity -> 'a * 'a
 val ft_comm :
      add:('comm -> 'comm -> 'comm)
   -> scale:('comm -> 'scalar -> 'comm)
-  -> endoscale:('comm -> 'c -> 'comm)
+  -> endoscale:'c
```

Comment:
Revert, the more descriptive type is better (even though the argument is unused).